### PR TITLE
[2/5] Visualizations (merge after 1/5)

### PR DESCRIPTION
**Pull request by ProfFan**

### CODE REVIEW

**Comment by dellaert** [comment]
I’ll review after 1/5 is merged as this PR seems cumulative

File: Sources/BeeTracking/Visualizations.swift
Comment (dellaert) [inline Sources/BeeTracking/Visualizations.swift:74]: document, and all arguments

Comment (dellaert) [reply Sources/BeeTracking/Visualizations.swift:74]: Maybe even make LikelihoodModel right away :-) Or better: make an extension for TrackingLikelihoodModel?

File: Sources/BeeTracking/Visualizations.swift
Comment (dellaert) [inline Sources/BeeTracking/Visualizations.swift:119]: document, extension for TrackingLikelihoodModel?

File: Sources/BeeTracking/Visualizations.swift
Comment (dellaert) [inline Sources/BeeTracking/Visualizations.swift:145]: make a flagged option

File: Sources/BeeTracking/Visualizations.swift
Comment (dellaert) [inline Sources/BeeTracking/Visualizations.swift:158]: document blocks

**Review by dellaert — APPROVED** [review]

File: Sources/BeeTracking/Visualizations.swift
Comment (ProfFan) [reply Sources/BeeTracking/Visualizations.swift:74]: Yeah I think I can try make that happen before merging

File: Sources/BeeTracking/Visualizations.swift
Comment (ProfFan) [reply Sources/BeeTracking/Visualizations.swift]: done

File: Sources/BeeTracking/Visualizations.swift
Comment (ProfFan) [reply Sources/BeeTracking/Visualizations.swift]: done

**Comment by ProfFan** [comment]
I'll merge to unblock :)

**Pull request merged**
